torch/_dynamo/compiled_autograd.py

```diff
@@ -421,6 +435,13 @@ def runtime_wrapper(compiled_fn, inputs, sizes, scalars, hooks):
             finally:
                 in_compiled_autograd_region = False
 
+        get_chromium_event_logger().log_event_end(
```

Comment:
Is there a chance the code between the start/end events fails or throws an exception? You might want this in a finally clause, or better yet, to use dynamo_timed directly